### DISCUSSION TITLE
Automatically increase patch-number

### DISCUSSION THREAD

**Discussion by jzwar** [tataratat/splinepy#190] [Ideas]
Versioning: `Major.Minor.Patch`

ImO we should automatically increase the Patch number, so new features and bug fixes are automatically uploaded to pypi-repo.

Is there a github action that can do this?
What do you think?
@j042 @clemens-fricke @danielwolff1

**Comment by clemens-fricke** [comment]
You could add an action to the build system/commit hook.

This seems to be working towards the same problem.
https://stackoverflow.com/questions/71740554/git-hook-to-increment-version-number-on-merge-to-master

The problem I see is that I am not sure how to then also use the system to push major/minor releases.

**Comment by j042** [comment]
I won't do automatic patch number increase. We could consider automatic dev version bump
